database/migrations/2016_08_18_152505_create_members_table.php

```diff
@@ -12,17 +12,15 @@ class CreateUsersTable extends Migration
      */
     public function up()
     {
-        Schema::create('users', function (Blueprint $table) {
+        Schema::create('members', function (Blueprint $table) {
```

Comment:
To be honest, I only realized how the migrations work after I changed these kinds of things. By now I think it might be wise to do it all over, to have it clean and since we are not really in a production stage anyway. I am happy to hear others thought about this issue.